_data/projects/elliotForWater.yml

```
@@ -0,0 +1,13 @@
+name: Elliot For Water
+desc: Elliot For Water is a search engine that donates water with your searches
+site: https://ellitoforwater.com/
+tags:
+- water
+- search-engine
+- humanitarian
+- ethical
+- open-source
+- nextjs
+upforgrabs:
+  name: good first issues
+  link: https://github.com/ElliotForWater/elliotforwater.com/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

:art: a more succinct link

```suggestion
  link: https://github.com/ElliotForWater/elliotforwater.com/labels/good%20first%20issue
```

---

ah, thanks @shiftkey